src/Drupal/ServiceMap.php

```diff
@@ -55,6 +55,9 @@ public function setDrupalServices(array $drupalServices): void
             );
             $deprecated = $serviceDefinition['deprecated'] ?? null;
             if ($deprecated) {
+                if (is_array($deprecated) && !empty($deprecated['message'])) {
```

Comment:
Use `isset` to check for the key

Reply:
Thanks, I have added the commit, but github is stalled "processing updates".